network/vif-route-qubes

```diff
@@ -120,8 +130,7 @@ if [ "${ip}" ] ; then
 fi
```

Comment:
IMO the hook should be called even if IP isn't specified. In that case, the argument should be empty.